lib/utils/date_time_extension.dart

```diff
@@ -28,9 +28,10 @@ extension DateTimeExtension on DateTime {
 
   /// Checks if two DateTimes are close enough to belong to the same
   /// environment.
-  bool sameEnvironment(DateTime prevTime) {
-    return millisecondsSinceEpoch - prevTime.millisecondsSinceEpoch <
-        1000 * 60 * minutesBetweenEnvironments;
+  bool sameDay(DateTime prevTime) {
```

Comment:
Suggestion: You can use func `DateUtils.isSameDay(date1, date2)` of `material` flutter package

Reply:
Thanks